cloudlift/session/session_creator.py

```diff
@@ -16,8 +18,14 @@ def __init__(self, name, environment):
     self.sts_client = get_client_for("sts", self.environment)
 
   def start_session(self, mfa_code):
-    mfa.do_mfa_login(mfa_code, get_region_for_environment(self.environment))
-    target_instance = self._get_target_instance()
+    user_id = (self.sts_client.get_caller_identity()['Arn'].split("/")[0]).split(":")[-1]
```

Comment:
Wrap in exception handler and raise UnrecoverableException if aws is not properly setup.